ui/components/app/confirm-page-container/confirm-page-container-header/confirm-page-container-header.component.js

```diff
@@ -64,7 +75,23 @@ export default function ConfirmPageContainerHeader({
             </span>
           </div>
         )}
-        {isFullScreen ? null : <NetworkDisplay />}
+        {isFullScreen ? null : (
+          <PickerNetwork
+            label={
+              networkType === NETWORK_TYPES.RPC
+                ? nickname ?? t('privateNetwork')
+                : t(getNetworkLabelKey(networkType))
+            }
+            backgroundColor={BackgroundColor.transparent}
+            borderColor={BorderColor.borderMuted}
+            iconProps={{ display: 'none' }}
```

Comment:
Let's use the design system enum helpers here 
```
import {
  BackgroundColor,
  BorderColor,
  Display
} from '../../../../helpers/constants/design-system';
```
```suggestion
            iconProps={{ display: Display.None }}
```

Reply:
updated